engine/src/main/java/org/terasology/world/block/family/AbstractBlockFamily.java

```diff
@@ -29,13 +32,23 @@
     private BlockUri uri;
     private Set<String> categories = Sets.newHashSet();
 
-    public AbstractBlockFamily(BlockUri uri, Iterable<String> categories) {
-        this.uri = uri;
+    public AbstractBlockFamily(BlockFamilyDefinition definition, BlockShape shape, BlockBuilderHelper blockBuilder) {
+    }
+
+    public AbstractBlockFamily(BlockFamilyDefinition blockFamilyDefinition, BlockBuilderHelper blockBuilderHelper) {
+    }
+
+    protected void setCategory(Iterable<String> categories) {
         for (String category : categories) {
             this.categories.add(category.toLowerCase(Locale.ENGLISH));
         }
     }
 
+    protected void setBlockUri(BlockUri uri) {
```

Comment:
Seems odd that this property is mutable now, but I assume it's a plumbing thing

Reply:
❓